### PR TITLE
[FSPE-5109] Add support of Sub-resource integrity to all Chi assets

### PR DESCRIPTION
**Pull request by janibolkvadze**
@dani-cl-madrid

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://02fd796b6cd47d53f227fcec782eddbebf7ed0a2-chi-pr.enterprise.ctl.io" target="_blank"> https://02fd796b6cd47d53f227fcec782eddbebf7ed0a2-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1814/ for more details.

File: package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "test-ce": "gulp test:custom-elements",
     "test:e2e": "gulp test:e2e",
     "test:e2e:interactive": "gulp test:e2e:interactive",
-    "echo": "gulp --version"
+    "echo": "gulp --version",
+    "sri": "gulp sri"
```
Comment (dani-cl-madrid) [inline package.json]: remove this line, please

**Review by dani-cl-madrid — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1818/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e67793679d8fe2a848384ef58db1a0ffebc0fcc2-chi-pr.enterprise.ctl.io" target="_blank"> https://e67793679d8fe2a848384ef58db1a0ffebc0fcc2-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1819/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e67793679d8fe2a848384ef58db1a0ffebc0fcc2-chi-pr.enterprise.ctl.io" target="_blank"> https://e67793679d8fe2a848384ef58db1a0ffebc0fcc2-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1820/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://e67793679d8fe2a848384ef58db1a0ffebc0fcc2-chi-pr.enterprise.ctl.io" target="_blank"> https://e67793679d8fe2a848384ef58db1a0ffebc0fcc2-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1821/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://0fae5e1cbc7128310bdf266b81c9ad9936e90da1-chi-pr.enterprise.ctl.io" target="_blank"> https://0fae5e1cbc7128310bdf266b81c9ad9936e90da1-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1822/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-fvvdkc, https://ops.elasticbox.com/services/instances/i-chayxd, https://ops.elasticbox.com/services/instances/i-mx45rc

**Pull request closed without merging**